index.js

```diff
@@ -1,9 +1,37 @@
 /* jshint node: true */
 'use strict';
 
+var buildMap = require('./lib/build-imports-map');
+var scope = require('./lib/broccoli-scope-scss-imports');
+var addMap = require('./lib/broccoli-add-imports-map');
+var mergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
   name: 'untitled-ui',
-  
+
+  setupPreprocessorRegistry: function(type, registry) {
+    if (type !== 'self') { return; }
+
+    var map = buildMap(registry.app.project.root, {
+      include: ['components/**/*.scss']
+    });
```

Comment:
this isn't the best because the map is only built at the beginning of the build, so if you add or remove files you need to restart the server :( not sure how to build up the map every time a css file changes _and_ thread the map threaded to the build